test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -63,14 +67,19 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 		profile, err = profiles.GetByNodeLabels(nodeLabel)
 		Expect(err).ToNot(HaveOccurred())
 		klog.Infof("using profile: %q", profile.Name)
-		performanceMCP, err = mcps.GetByProfile(profile)
-		Expect(err).ToNot(HaveOccurred())
-		klog.Infof("using performanceMCP: %q", performanceMCP)
-
-		// Verify that worker and performance MCP have updated state equals to true
-		for _, mcpName := range []string{testutils.RoleWorker, performanceMCP} {
-			mcps.WaitForCondition(mcpName, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+		if !hypershift.IsHypershiftCluster() {
```

Comment:
I think we can use the `profilesupdate.WaitForTuningUpdating` and `profilesupdate.PostUpdateSync` here. 
Not sure we need the to check the worker label as well

Reply:
Here we are not doing any profile changes, we are checking mcp is updated, before tests are run.